goes16ci/models.py

```diff
@@ -81,7 +82,7 @@ def __init__(self, min_filters=16, filter_growth_rate=2, filter_width=5, min_dat
                  hidden_activation="relu", output_activation="sigmoid",
                  pooling="mean", use_dropout=False, dropout_alpha=0.0,
                  data_format="channels_last", optimizer="adam", loss="mse", leaky_alpha=0.1, metrics=None, 
-                 learning_rate=0.001, batch_size=1024, epochs=10, verbose=0, sgd_momentum=0.99):
+                 learning_rate=0.001, batch_size=1024, epochs=10, verbose=0, sgd_momentum=0.99,trail=None):
```

Comment:
I am assuming trail=None should be trial=None. It doesn't look like trial is used in the `__init__` method.